tests/qos/test_buffer.py

```diff
@@ -2481,15 +2481,8 @@ def _check_port_buffer_info_and_return(duthost, table, ids, port, expected_profi
         else:
             if is_mellanox_device(duthost):
                 buffer_items_to_check = buffer_items_to_check_dict["down"][key_name]
-            elif is_broadcom_device(duthost) and (asic_type in ['td2', 'td3'] or speed <= '10000'):
```

Comment:
Why remove `is_broadcom_device` condition? We also have other platform such as `marvell` , seems like this change will also cause skipping happening on other platforms, not just broadcom.

Reply:
The special handling is needed only for mellanox. buffermgr will not attach any profiles for admin down ports on other platforms

Reply:
@lipxu  could you update your PR title to `Skip all admin down ports for non-Mellanox device`?

Reply:
Sure, @Blueve , have updated the title.
Updated the commit last time, but forgot to update the title, sorry for the confusion.